### PR TITLE
Update version: GitButler.GitButler version 0.12.9

### PR DESCRIPTION
**Pull request by damn-good-b0t**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/162317)

### CODE REVIEW

File: manifests/g/GitButler/GitButler/0.12.9/GitButler.GitButler.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+PackageIdentifier: GitButler.GitButler
+PackageVersion: 0.12.9
+PackageLocale: en-US
+Publisher: GitButler
+PublisherUrl: https://gitbutler.com
+PrivacyUrl: https://app.termly.io/policy-viewer/policy.html?policyUUID=a001c8b7-505b-4eab-81e3-fcd1c72bdd79
+PackageName: GitButler
+PackageUrl: https://gitbutler.com
+License: Functional Source License, Version 1.1, MIT Future License
+LicenseUrl: https://github.com/gitbutlerapp/gitbutler/blob/7d01a537d49e0219773dacb1655f6a7a58f5f355/LICENSE.md
```
Comment (mdanish-kh) [inline manifests/g/GitButler/GitButler/0.12.9/GitButler.GitButler.locale.en-US.yaml:13]:
```suggestion
LicenseUrl: https://github.com/gitbutlerapp/gitbutler/blob/master/LICENSE.md
```

File: manifests/g/GitButler/GitButler/0.12.9/GitButler.GitButler.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+Copyright: Copyright 2023-2024 GitButler Inc
+CopyrightUrl: https://github.com/gitbutlerapp/gitbutler/blob/7d01a537d49e0219773dacb1655f6a7a58f5f355/LICENSE.md
```
Comment (mdanish-kh) [inline manifests/g/GitButler/GitButler/0.12.9/GitButler.GitButler.locale.en-US.yaml:15]:
```suggestion
CopyrightUrl: https://github.com/gitbutlerapp/gitbutler/blob/master/LICENSE.md
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
It's better to use master branch URLs instead of specific commits, so users see the latest file contents when they visit this URL

**Comment by a-mnich** [comment]
> It's better to use master branch URLs instead of specific commits, so users see the latest file contents when they visit this URL

@mdanish-kh  Yes, but that's only true for files where you actually want them to be up-to-date. For a specific version of a package, usually the license that was in the repository at that point in time applies. When the package is re-licensed in a later version, the old license still applies and therefore I think the manifest should point to a specific commit.

**Comment by mdanish-kh** [comment]
@a-mnich Generally package updates here are made through tools that automatically generate the manifest for you (wingetcreate as an example). These tools generally don't have the ability to dynamically fetch the latest upstream commit's LicenseUrl for every new update. What you mention is desirable, and in that case, the `LicenseUrl` here should actually be `https://github.com/gitbutlerapp/gitbutler/blob/release/0.12.9/LICENSE.md` instead. The URL in the PR is the same carried over from previous manifests, which is exactly my point that the tools available don't allow you to easily change this URL every update. It is better to just go with a static latest URL, to add less friction in the contribution / PR generation process.

If the license changes, one can easily write a script to go through the older manifests and update `LicenseUrl` to some commit / release branch before the license change was made. However, license changes are generally very rare.

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**